blockchain/fundamentals/ethereum-i/ethereum-and-ether.md

```diff
@@ -12,32 +11,47 @@ type: normal
 category: must-know
 
 tags:
-
   - introduction
 
   - deep
 
+links:
+  - '[Ethereum](https://www.ethereum.org/){website}'
+  - '[Ethereum source code](https://github.com/ethereum/go-ethereum){website}'
+  - '[Guide to Casper Protocol](https://blockgeeks.com/guides/ethereum-casper/){article}'
 ---
+
 # Ethereum and Ether
 
 ---
+
 ## Content
 
+**Ethereum** is the blockchain that changed how the game is played. Before its creation, most cryptocurrencies were the cryptographic equivalent of fiat currencies. Once it was launched, cryptocurrencies started to represent something: a token through which people could interact with a given platform.
+
+Ethereum is a distributed supercomputer. Its main feature is the ability to store and run applications. It was created by _Vitalik Buterin_, back then only 21 years old. The code is open-source and can be found on GitHub.
 
+Applications are run using blockchain's turing complete virtual machine: Ethereum Virtual Machine or EVM. Every node in the network will therefore run the code you deploy on the blockchain. How do you make sure people don't run malicious code or infinite loops?
```

Comment:
how then

Reply:
it is supposed to be linked with the next paragraph